frontend/src/components/empty-event/index.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component, PropTypes } from 'react'
+
+export default class EmptyEvent extends Component {
+  static get propTypes () {
+    return {
+      events: PropTypes.array.isRequired,
+      isFetchingEvents: PropTypes.bool.isRequired
+    }
+  }
+
+  render () {
+    return (
+      this.props.events.length === 0 &&
+        !this.props.isFetchingEvents &&
```

Comment:
or even remove the entire conditioning to the parent, and make this a no-props component ;-)